Cython/Compiler/Lexicon.py

```diff
@@ -159,184 +157,184 @@ def prefixed_digits(prefix, digits):
 # cpython 3.12.0a7+ (heads/master:4cd1cc843a, Apr 11 2023, 10:32:26) [GCC 11.3.0]
 
 unicode_start_ch_any = (
-    u"\u005f\u00aa\u00b5\u00ba\u02ec\u02ee\u037f\u0386\u038c\u0559\u06d5"
```

Comment:
These are auto-generated so will come back (unless you change the auto-generator)

Reply:
I'll update the generator.